tests/test_basic.py

```
@@ -121,7 +122,7 @@ def test_IsNone_WHEN_given_corner_case_input(self):
 				(_ignored_code, test_fixture) = tst_dispatch.useTool(tst_in)
 				self.assertIsNone(
 					test_fixture,
-					str("""multicast.McastDispatch().useTool({}) == ERROR""").format(str(tst_in))
+					f"multicast.McastDispatch().useTool({str(tst_in)}) == ERROR"
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)